app/pkg/dbx/migrate.go

```diff
@@ -140,3 +144,27 @@ func getLastMigration() (int, error) {
 
 	return int(lastVersion.Int64), nil
 }
+
+func getPendingMigrations(versions []int) ([]int, error) {
+	versionStr := ""
+	for _, version := range versions {
+		versionStr = versionStr + "," + strconv.Itoa(version)
+	}
+	versionStr = versionStr[1:]
+
+	rows, err := conn.Query("SELECT version FROM migrations_history WHERE version IN (" + versionStr + ")")
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
I think we need a defer rows.Close here (appreicate the other functions don't but I think we should do that here)